app/src/main/java/com/example/dynamicdiet/MainActivity.kt

```diff
@@ -84,9 +80,13 @@ fun SimpleFilledTextFieldSample() {
             .padding(15.dp)
     ) {
         Column(
+            modifier = Modifier
+                .fillMaxWidth()
+                .background(MaterialTheme.colors.background)
+                .padding(15.dp),
             verticalArrangement = Arrangement.Center
         ) {
-            TextField(
+            OutlinedTextField(
```

Comment:
I would suggest to use Outlined TextField. The label is on the top of the border, nice and clear.